cmd/atlascmd/migrate/migrate.go

```diff
@@ -65,12 +125,19 @@ func (r *EntRevisions) WriteRevisions(ctx context.Context, rs migrate.Revisions)
 			SetOperatorVersion(rev.OperatorVersion).
 			SetMeta(rev.Meta)
 	}
-	return r.c.Revision.CreateBulk(bulk...).
+	return r.ec.Revision.CreateBulk(bulk...).
 		OnConflict(
 			sql.ConflictColumns(revision.FieldID),
 		).
 		UpdateNewValues().
 		Exec(ctx)
 }
 
+// Close closes the Atlas client connected to the revision schema (if any).
+func (r *EntRevisions) Close() {
+	if r.sc != nil {
+		r.sc.Close()
+	}
```

Comment:
In case of SQLite we use the existing sqlclient for the ent connection. Closing Ent in this case would result in the underlying DB for the atlas connection to be closed as well. This is the reason I have the `sc` and close that one (if present).

Reply:
👍